### DISCUSSION TITLE
What is the source for the catalogue?

### DISCUSSION THREAD

**Discussion by karypid** [prateekmedia/appimagepool#88] [General]
Hello,

Excuse my very basic question but I am not familiar with AppImage. As far as I know there is no "central" location for AppImage files so I was wondering how this utility creates its catalogue? How does one add an AppImage to this catalogue?

For example, I use Odysee.com's LBRY-desktop player which is published as an AppImage, but it is not available when I search with appimagepool. How can one add it to the catalogue of this utility so that I don't need to download LBRY from their website (but rather via appimagepool).

Thank you!

**Comment by prateekmedia** [comment]
We use AppImageHub as a catalogue.

Contribute new appimages from here: https://github.com/AppImage/appimage.github.io

**Reply by karypid** [reply]
> Contribute new appimages from here: https://github.com/AppImage/appimage.github.io

Thank you @prateekmedia

Is it possible to run appimagepool and have it read definitions from a github fork of the catalogue?

For example, I forked AppImage to https://github.com/karypid/appimage.github.io

I want to run appimagetool and have it read definitions from this branch:
[EDIT fixed URL]: https://github.com/karypid/appimage.github.io/tree/LBRY-desktop

I just want to test that what I added is parsed/processed correctly. Is that possible?

**Reply by prateekmedia** [reply]
Not currently, only method to do this is fork this app and change the url of feed.json which is difficult or you can just contribute to the upstream project they generally accept pull request.

**Reply by karypid** [reply]
Hey @prateekmedia 

Thanks for these pointers. I wanted to give this a go, but embarrassingly I can't seem to locate a `feed.json` file in the repo... Or maybe have you mistyped the filename?

**Reply by prateekmedia** [reply]
It is autogenerated that's why I said it would be hard.

You would need to setup github pages then go to the yourusername.github.io/appimage.github.io/feed.json

I don't know the procedure of that correctly
